setup.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+
+from setuptools import setup, find_packages
+
+setup(
+    name='tc_librato',
+    version="0.0.1",
+    description='Thumbor Librato extensions',
+    author='Peter Schröder, Sebastian Eichner',
+    author_email='peter.schroeder@jimdo.com, sebastian.eichner@jimdo.com',
+    zip_safe=False,
+    include_package_data=True,
+    packages=find_packages(),
+    install_requires=[
+        'jimdo-thumbor',
```

Comment:
@masom yeah, i tried that and it works. from what i've read `setup.py` has removed support for this.